src/modules/Common/components/Instances.tsx

```diff
@@ -4,12 +4,12 @@ import CardList from 'modules/Common/components/CardList';
 import CardTable from 'modules/Common/components/CardTable';
 import CardTableItem from 'modules/Common/components/CardTableItem';
 import { FiSearch as IconSearch } from 'react-icons/fi';
+import Image from 'next/image';
 import Link from 'next/link';
 import LinkIcon from 'modules/Common/components/LinkIcon';
 import React from 'react';
-import Image from 'next/image';
 import instancesData from '../../../../public/instances.json';
-import { kebabCase } from 'lodash';
+import slugify from 'slugify';
 import { useRouter } from 'next/router';
 import { useTranslation } from 'next-i18next';
```

Comment:
You need to rebase this branch and use

`import useTranslation from 'next-translate/useTranslation';`